website/src/components/Schema/CodeList.js

```diff
@@ -15,8 +15,8 @@ const ToggleMore = ({ children, onToggle }) => (
 
 const CodeList = ({ codeList = [], onToggle, isExpanded }) => {
   const maxEnumLength = 5;
-  const fullOutput = codeList.map(item => (
-    <p className={styles.fieldEnumValue}>
+  const fullOutput = codeList.map((item, i) => (
+    <p key={i} className={styles.fieldEnumValue}>
```

Comment:
if `item` is a string or number, it's actually better if we use it as `key` :) React essentially wants to have a unique identifier for the entry ;)